scanner/src/main/kotlin/PathScanner.kt

```diff
@@ -212,7 +212,7 @@ abstract class PathScanner(
         val missingArchives = mutableSetOf<Pair<Package, KnownProvenance>>()
 
         scanResults.forEach { (pkg, results) ->
-            results.forEach { (provenance, _, _) ->
+            results.forEach { (provenance, _, _, _) ->
```

Comment:
Nit: I believe this could be squashed to the previous commit, so that the same type of inspection hint is addressed in a single commit even if it occurs in multiple files.